### PR TITLE
LPS-198113 Tool tip is being displayed not displayed correctly

### PR DESCRIPTION
**Pull request by Artartur**
Related Issue: [LPS-198113](https://liferay.atlassian.net/browse/LPS-198113)

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @naraandrade,
I'm assigning this PR to you because it is frontend. Feel comfortable assigning to anybody from your squad :+1:

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Toggle.scss [hunk 1]
```diff
@@ -1,5 +1,10 @@
 @import 'atlas-variables';
 
-.lfr-objects__toggle-tooltip-icon {
-	color: $secondary-l1;
+.lfr-objects__toggle {
+	display: flex;
+	flex-direction: row;
```
Comment (igor-franca) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Toggle.scss:5]: Hi @Artartur, as the default value of flex-direction is row, could you check if this row is really necessary?

**Review by igor-franca — COMMENTED** [review]
Hey @Artartur! Thanks for this PR!

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Toggle.scss (hunk 1 above)
Comment (Artartur) [reply modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Toggle.scss]: You are right, I've changed it, thx for the review @igor-franca

**Comment by Artartur** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
After seeing the PR changes and analyzing the history of IC results, I can conclude that the failures of this test are not related to the changes.

**Comment by igor-franca** [comment]
ci:forward:force

**Pull request closed without merging**